docs/process_updating_mce_hypershift.md

```diff
@@ -0,0 +1,35 @@
+# Updating Hypershift
+
+The ACM is packaged as an OCM Addon and deploy to the service cluster with fleet manager.
+
+ACM includes the multicluster engine operator (MCE) which is the component that really packages the hypershift operator and it's related images. ACM provides the policy functionality, and MCE provides all the cluster lifecycle functionality.
+
+## Ways to Rollout
+
+There are four ways for fleet manager to upgrade the hypershift images.
+
+1. Hypershift image configmap
+
+   On a cluster with ACM 2.7.0/MCE 2.2.0 deployed, you have granular control over the hypershift image set. Using a configmap you can override the hypershift images.
+
+   see: https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/upgrading_hypershift_operator.md
+   
+2. Create a new ACM OCM Addon version
+
+   We build the ACM/MCE every 12 hours. Not all these builds pass bvt/svt. When they do, they can be promoted to OCM Addon. Currently, fleet manager deploys from our `beta` channel into `integration` and `staging`. The promotion is gated by approvals to MR against managed-tenant-bundles and managed-tenant.
+
+   * Right now, the rate of new ACM OCM Addon is on demand and manually, triggered when there is features that we need to make available to SD. This is basically the standard MT Addon update flow to the two MT repos. We have automation to create the MR for approval, just have to tweak.
```

Comment:
Currently a new version of ACM OCM Addon is built manually and on demand when there are features or bug fixes for SD. This is the standard MT Addon update flow to the two MT repos. There is automation to create the MR for approval but some tweak is required.